src/Drupal/Commands/core/QueueCommands.php

```diff
@@ -48,15 +52,13 @@ public function getQueueService(): QueueFactory
 
     /**
      * Run a specific queue by name.
-     *
-     * @command queue:run
-     * @aliases queue-run
-     * @param string $name The name of the queue to run, as defined in either hook_queue_info or hook_cron_queue_info.
-     * @validate-queue name
-     * @option time-limit The maximum number of seconds allowed to run the queue.
-     * @option items-limit The maximum number of items allowed to run the queue.
-     * @option lease-time The maximum number of seconds that an item remains claimed.
      */
+    #[CLI\Command(name: 'queue:run', aliases: ['queue-run'])]
+    #[CLI\Argument(name: 'name', description: 'The name of the queue to run, as defined in either hook_queue_info or hook_cron_queue_info.')]
+    #[CLI\Option(name: 'time-limit', description: 'The maximum number of seconds allowed to run the queue.')]
+    #[CLI\Option(name: 'items-limit', description: 'The maximum number of items allowed to run the queue.')]
+    #[CLI\Option(name: 'lease-time', description: 'The maximum number of seconds that an item remains claimed.')]
+    #[CLI\HookCustom(name: self::VALIDATE_QUEUE, arguments: ['name'])]
```

Comment:
Should we just make the parameters to this attribute be `name` and `argument` rather than `name` and `arguments`? All that we ever do with the arguments is implode with ` `; it seems more straightforward in this case to let the user define the string they want.

Reply:
I am just trying to be as native as possible to the command author. The fact that we implode later is a (slightly embarassing) implementation detail.

Reply:
Done